e2e-playwright/dashboard-cujs/dashboard-navigation.spec.ts

```diff
@@ -0,0 +1,243 @@
+        await expect.soft(filterEditButton).toHaveCount(1);
+        await expect.soft(filterEditButton).toBeVisible();
+      });
+
+      await test.step('1.2.Select a scope across multiple types of production entities', async () => {
+        await gotoDashboardPage({ uid: FIRST_DASHBOARD });
+
+        await openScopesSelector(page, testScopes);
+
+        const firstLevelScopes = testScopes[0].children!;
+        await expandScopesSelection(page, firstLevelScopes, testScopes[0].name);
+
+        const secondLevelScopes = firstLevelScopes[0].children!;
+        await expandScopesSelection(page, secondLevelScopes, firstLevelScopes[0].name);
+
+        const selectedScopes = [secondLevelScopes[0], secondLevelScopes[1]];
+        for (const scope of selectedScopes) {
+          await selectScope(page, scope);
+        }
+        await applyScopes(page, selectedScopes);
+
+        await expect
+          .soft(page.getByTestId('scopes-selector-input'))
+          .toHaveValue(`${secondLevelScopes[0].title!}, ${secondLevelScopes[1].title!}`);
+        // Verify that scope filter is injected
+        const filterEditButton = page.locator(
+          `[aria-label="Edit filter with key ${secondLevelScopes[0].filters![0].key}"]`
+        );
+        // Ensure exactly one exists and it's visible
+        await expect.soft(filterEditButton).toHaveCount(1);
+        await expect.soft(filterEditButton).toBeVisible();
+      });
+
+      await test.step('1.3.View and select a scope configured by any team', async () => {
+        await gotoDashboardPage({ uid: FIRST_DASHBOARD });
+
+        await openScopesSelector(page, testScopes);
+
+        const firstLevelScopes = testScopes[2].children!;
+        await expandScopesSelection(page, firstLevelScopes, testScopes[0].name);
+
+        const secondLevelScopes = firstLevelScopes[0].children!;
+        await expandScopesSelection(page, secondLevelScopes, firstLevelScopes[0].name);
+
+        const selectedScopes = [secondLevelScopes[0]];
+
+        for (const scope of selectedScopes) {
+          await selectScope(page, scope);
+        }
+
+        await applyScopes(page, []);
+
+        await expect.soft(page.getByTestId('scopes-selector-input')).toHaveValue(secondLevelScopes[0].title!);
+      });
+
+      await test.step('1.4.View and select a recently viewed scope', async () => {
+        // this step depends on the previous ones because they set recent scopes
+        await gotoDashboardPage({ uid: FIRST_DASHBOARD });
+
+        await openScopesSelector(page, testScopes);
+
+        await page.getByTestId('scopes-selector-recent-scopes-section').click();
+
+        const scopes = testScopes[0].children![0].children!;
+
+        await page.locator(`button:has(span:text("${scopes[0].title!}, ${scopes[1].title!}"))`).click();
+
+        await expect
+          .soft(page.getByTestId('scopes-selector-input'))
+          .toHaveValue(`${scopes[0].title!}, ${scopes[1].title!}`);
```

Comment:
I can see this assertion failing when running the test. Do you see that too @mdvictor ? Similarly I see the same failure in 1.2 assertion.